README.md

```diff
@@ -228,16 +228,16 @@ For example:
 chmod 400 id_rsa && \
 ssh-add $(pwd)/id_rsa
 
-ssh vcap@<concourse_ip>
+ssh vcap@<deployer_concourse_ip>
```

Comment:
Given that we're assigning DNS names to the deployer_concourse, might it be easier to use the name rather than the IP here?